src/Axe/Managers/CompilerManager.cs

```diff
@@ -265,10 +276,105 @@ public CompilerManager(AxeDbContext context, INodeServices nodeServices) : base(
                 return new Tuple<CodeBlockResult, string[]>(CodeBlockResult.Error, new string[] { exception.Message });
             }
         }
-        
+
+        private async Task<Tuple<CodeBlockResult, string[]>> InterpretPython(CodeBlockSolveVm model)
+        {
+            try
+            {
+                CodeBlock codeBlock = context.CodeBlock
+                    .Include(cb => cb.TestCases)
+                    .First(cb => cb.Id == model.Id);
+                List<string> failedTestCases = new List<string>();
+
+                string pyFilePath = Path.GetTempPath() + Path.GetRandomFileName().Replace(".", "") + ".py";
+
+                //ProcessStartInfo startInfo = new ProcessStartInfo("C:\\Users\\andrei.shilkin\\AppData\\Local\\Programs\\Python\\Python36-32\\python.exe");
+                ProcessStartInfo startInfo = new ProcessStartInfo(Environment.GetFolderPath(Environment.SpecialFolder.UserProfile) + "\\AppData\\Local\\Programs\\Python\\Python36-32\\python.exe");
+                startInfo.Arguments = pyFilePath;
+                startInfo.UseShellExecute = false;
+                startInfo.CreateNoWindow = true;
+                startInfo.RedirectStandardOutput = true;
+                startInfo.RedirectStandardError = true;
+
+                for (int i = 0; i < codeBlock.TestCases.Count; i++)
```

Comment:
can't find usegaes for `i` variable. why not use `foreach` on `codeBlock.TestCases`?